web/gatsby-config.js

```diff
@@ -66,5 +66,14 @@ module.exports = {
         trackingId: "UA-47629346-5",
```

Comment:
@moul we have this GA plugin and config, we should keep this and replicate on the www?

Reply:
Keep it for now, we’ll remove it in few weeks

No need to replicate on www